samples/subsys/ipc/ipc_service/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _IPC_Service_sample:
+
+IPC Service sample Application
+##############################
```

Comment:
```suggestion
IPC Service
###########
```
when adding a sample or library, do not mention sample or library in the title of the page (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#titles)

Reply:
Thanks. Fixed.